views/address.tmpl

```diff
@@ -13,6 +13,20 @@
                 <h4>Address</h4>
                 <p class="mono">{{.Address}}</p>
 
+                {{$Transactions := len .Transactions}}
+                {{$Unconfirmed := 0}}
+                {{$CurrentBalance := 0}}
+                {{range .Transactions}}
+                    {{if eq .Time 0}}
+                        {{$Unconfirmed +1}}
+                    {{end}}
+                {{end}}
+                <p>Transactions:   {{$Transactions}}</p>
+                <p>Current Balance: {{$CurrentBalance}}</p>
+                <p>Unconfirmed: {{$Unconfirmed}}</p>
+                {{if eq $Transactions 2000}}
+                    Currently only the first 2000 transactions are being used.
```

Comment:
I think the number is more like 700 now, but I don't love putting a magic number here.

Reply:
I also wasn't sure it was still 2000, because i kept getting a max high of 500. I could change this now and wait for the new DB before anything else here. Just confirming if it's 700 first?

Reply:
yup, looks to be 500 actually, like you said: https://github.com/dcrdata/dcrdata/blob/b98c0b9bbbf7d0e3a7288e80f33f55a842d1fed1/explorer/explorer.go#L31